packages/stack-shared/src/known-errors.tsx

```diff
@@ -1378,6 +1378,17 @@ const EmailRenderingError = createKnownErrorConstructor(
   (json: any) => [json.error] as const,
 );
 
+const ThemeWithNameAlreadyExists = createKnownErrorConstructor(
+  KnownError,
+  "THEME_WITH_NAME_ALREADY_EXISTS",
+  (name: string) => [
+    400,
+    `A theme with the name "${name}" already exists.`,
+    { name },
+  ] as const,
+  (json: any) => [json.name] as const,
+);
+
```

Comment:
Non-unique display names should be fine. We should only care about non-unique IDs